moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -338,6 +343,7 @@ export default function TheMap({
         onClick={onClick}
         cursor={cursor}
         mapStyle={basemaps[basemapKey].mapStyle}
+        onLoad={checkIfMapHasPlaceHolderLayers}
```

Comment:
ok, so the map load event triggers the layer check and `hasPlaceHolderLayers` state update—cool.

assuming this `load` event only ever fires once, is there a situation where `checkIfMapHasPlaceHolderLayers` would not set `hasPlaceHolderLayers` to `true`? or to put it another way, would it be more clear to use a generic `hasMapLoaded` state instead?

Reply:
I did some testing, and this is a good catch. I wrote the code to specifically check for that layer, but that check isn't necessary for the errors to not occur. It's misleading and makes this look more complex than it is.

I'll make this more general and add comments about why the layers are waiting on the map to fully load before rendering component layers. ✅